### ISSUE TITLE
benchmark with more files 

### ISSUE THREAD

**Issue by sundy-li** [tustvold/access-log-bench#5]
In my opinion, duckdb runs slow because it processes multiple rowgroups of a single file serially. This test could try testing multiple files to compare the differs.

**Comment by tustvold** [comment]
The same is true of DataFusion?

**Comment by sundy-li** [comment]
> The same is true of DataFusion?

Oh, I did not know that DataFusion read a single file `rowgroup` by `rowgroup`, I thought it was on parallel.

**Comment by tustvold** [comment]
Doing so would change the ordering of the file, and DataFusion isn't yet smart enough to determine that downstream operators are order agnostic and push this into ParquetExec in the form of a `FileRange` predicate. You can manually construct such a physical plan, but it won't do it automatically.

In practice we normally have the reverse problem of too much parallelism in the parquet scan, as a result of lots of files, adding memory pressure and contention, and so this hasn't been a major priority for us

**Comment by peterinfra** [comment]
Duckdb supports file globbing to execute queries on multiple files in parallel (would be a better comparison for time partitioned log files). Please add that to the benchmark.

**Comment by tustvold** [comment]
I would be happy to review a PR adding this, however, as all three systems support this and it is an embarrassingly parallel workload, I don't foresee it showing anything further
